### PR TITLE
Indicate duplicate string

### LINKED ISSUE #911: qs: indicate duplicate strings
for example:
```
string 1
string 2
string 3
string 1  << duplicate
string 4
````

this way unique strings can quickly be identified but no data is hidden by default

### PR DESCRIPTION
**Pull request by ooprathamm**
Closes #911
 
#duplicate tag + Mute
* Sample output
image

### CODE REVIEW

**Comment by williballenthin** [comment]
This is a great start. I wonder if we should not tag the first instance of the string, only the second+ as duplicate. Otherwise important, but duplicated, strings would all get muted every time. 

Do you think this is feasible @ooprathamm?

File: floss/qs/main.py [hunk 1]
```diff
@@ -651,13 +651,21 @@ def tag_strings(self, taggers: Sequence[Tagger]):
         this can be overridden, if a subclass has more ways of tagging strings,
         such as a PE file and code/reloc regions.
         """
+        string_counts = {}
```
Comment (mr-tz) [inline floss/qs/main.py:654]: can use defaultdict(int) here

**Review by mr-tz — COMMENTED** [review]
I think like this is good. How would you otherwise show the << data? In a new column?

**Comment by ooprathamm** [comment]
I was considering rendering count on the console directly ahead of strings, adding a new column for duplicate strings would be excessive. However, appending the count to the strings disrupts other functionalities. Hence, I suggest proceeding with the "#duplicate" tag.

File: floss/qs/main.py [hunk 2]
```diff
@@ -651,13 +652,21 @@ def tag_strings(self, taggers: Sequence[Tagger]):
...
+        
         tagged_strings: List[TaggedString] = []
+
         for string in self.strings:
             # at this moment, the list of strings contains only ExtractedStrings.
             # this routine will transform them into TaggedStrings.
             assert isinstance(string, ExtractedString)
             tags: Set[Tag] = set()
 
+            string_counts[string.string] = string_counts.get(string.string, 0) + 1
```
Comment (mr-tz) [inline floss/qs/main.py:665]:
```suggestion
            string_counts[string.string] += 1
```

**Comment by mr-tz** [comment]
thanks!

File: floss/qs/main.py (hunk 1 above)
Comment (williballenthin) [reply floss/qs/main.py]: or `collections.Counter`

**Pull request merged**